plugins/modules/datastore.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["stableinterface"],
+    "supported_by": "certified",
+}
+
+DOCUMENTATION = '''
+module: datastore
+author:
+  - Manca Bizjak (@mancabizjak)
+  - Tadej Borovsak (@tadeboro)
+short_description: Manage Sensu external datastore
+description:
+  - Activate or deactivate external datastore.
```

Comment:
```suggestion
  - Add or remove external datastore provider.
```

Reply:
These are remnants of my previous incarnations of this module where I blindly assumed that external datastore is either enabled or disabled. In theory, things are a bit more flexible, but for all practical purposes, my initial assumption was not completely wrong ;)